src/css/betterttv.css

```diff
@@ -157,8 +153,16 @@ input.text:focus,select:focus,textarea:focus {
    text-decoration: none;
 }
 
-.ember-chat .badges, .ember-chat .mod-icons {
-    font-size: 0;
+.ember-chat .badges span:last-child .badge {
```

Comment:
the last time you messed with badges you caused a lot of alignment issues when there were:

1) no mod icons
2) no badges
3) no timestamps
4) mix-match 1-3